configuration/admin_lock_pipelines.md

```diff
@@ -3,20 +3,49 @@ description: Ensure only one instance of a GoCD pipeline can run at the same tim
 keywords: GoCD congfiguration, continuous delivery pipeline, stage, locked pipeline, pipeline scheduling, pipeline locking
 ---
 
-# Ensure only one instance of a GoCD pipeline can run at the same time
+# Pipeline locking: Ensure only one instance of a GoCD pipeline can run at the same time
 
 Sometimes you want to ensure that only a single instance of a pipeline can run at a time. This is important if the stages of a pipeline are interrelated. For example the first stage may set up an environment that is used by the next stage in the pipeline.
 
 If a pipeline is locked then GoCD will not allow any other instance of that pipeline to be scheduled until the currently running one has been completed.
 
-To enable locking from the Config UI, navigate to the General Options section of pipeline.
+To enable locking from the Config UI, navigate to the Pipeline Settings section of a pipeline. There are three options for locking:
 
-![](../resources/images/pipeline_locking.png)
 
-To enable locking from the Config XML set the isLocked attribute to true
+## Option 1: Unlock when finished
+
+This option enables a single instance of the pipeline to run at a time, but does not lock the pipeline when it is
+finished. The lock is active only till the pipeline is in progress, to ensure that only a single instance runs. When the
+pipeline finishes, either because of the failure of a stage or the success of the last stage, the pipeline is
+automatically unlocked. The pipeline is also unlocked if it reaches a manual stage.
+
+![Pipeline locking behavior - Unlock when finished](../resources/images/pipeline_lock_behavior_01_unlock_when_finished.png)
+
+
+## Option 2: Lock on failure
+
+This option enables a single instance of the pipeline to run at a time, and the pipeline gets locked if it fails. The
```

Comment:
> and the pipeline gets locked if it fails.

Wondering if it is right to call that `pipeline gets locked` since the pipeline is already locked. Should we say it will remain locked on failure.

Reply:
Ok. Will make this change.